Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -371,13 +371,13 @@ GLuint ProgramShaderCache::CompileSingleShader(GLuint type, const char* code)
 
 		if (compileStatus != GL_TRUE)
 		{
-			PanicAlert("Failed to compile %s shader!\nThis usually happens when trying to use Dolphin with an outdated GPU or integrated GPU like the Intel GMA series.\n\nIf you're sure this is Dolphin's error anyway, post the contents of %s along with this error message at the forums.\n\nDebug info (%s, %s, %s):\n%s",
-				type == GL_VERTEX_SHADER ? "vertex" : type==GL_FRAGMENT_SHADER ? "pixel" : "geometry",
-				filename.c_str(),
-				g_ogl_config.gl_vendor,
-				g_ogl_config.gl_renderer,
-				g_ogl_config.gl_version,
-				infoLog);
+			PanicAlertT("Failed to compile %s shader!\n"
+			            "This usually happens when trying to use Dolphin with an outdated GPU or integrated GPU like the Intel GMA series.\n\n"
+			            "If you're sure this is Dolphin's error anyway, post the contents of %s along with this error message at the forums.\n\n"
+			            "Debug info (%s, %s, %s):\n%s",
+			            type == GL_VERTEX_SHADER ? "vertex" : type==GL_FRAGMENT_SHADER ? "pixel" : "geometry",
```

Comment:
If we're going to translate this, we probably want to translate the words "vertex", "pixel", and "geometry" as well.